rfc/1/index.md

```diff
@@ -1,79 +1,134 @@
+    - [Review](./reviews/1/index)
+*   - Matthew Hartley
+    - mrmh2
+    - EMBL-EBI
+    - 2024-03-05
+    - [Review](./reviews/3/index)
+*   - Wouter-Michiel Vierdag, Luca Marconato
+    - melonora, LucaMarconato
+    - EMBL
+    - 2024-01-13 
+    - [Comment](./comments/1/index)
+*   - Matt McCormick
+    - thewtex
+    - ITK
+    - 2024-01-09 
+    - [Comment](./comments/2/index)
+*   - Josh Moore
+    - joshmoore
+    - German BioImaging
+    - 2024-08-29 
+    - [Response](./responses/1/index)
+```
 
 ## Overview
 
 This RFC defines the high-level decision making process for changes within the
 NGFF community. These changes are defined in "Request for Comments" (RFCs) and
 therefore this RFC is self-referential: it is following the process that it
-itself defines. The overall goal of the process is to make clear how decision
-making works with a focus on both speed of development and clarity. It should
-be clear after reading the RFC which stakeholder (author, reviewer, editor,
-etc.) is responsible for moving decisions along and how much time the
-community can expect that decision to take. Not all decisions in the NGFF
-community need to follow the RFC process. Clarifications, corrections, and
-numerous other changes will proceed following the current GitHub workflow.
-However, when decisions reach a certain scale, including significant
-specification changes but also changes to the community process itself, RFCs
-will provide a mechanism for managing the process.
+itself defines.
 
 ## Background
 
 Growing interest in the NGFF format has also led to increased participation in
 the specification process. As such, reaching a consensus for all decisions has
-become more difficult. The current approach defined post facto in RFC-0
+become more difficult. The current approach defined _post facto_ in RFC-0
 follows a full consensus model. Through community meetings and pull requests,
 it was expected that all parties agree before a specification change was
-considered. This made it especially difficult both for **Authors** as well as
-**Reviewers** to know when a suggested change was adopted since PR comments
-could re-open a discussion. It also left a significant burden on **Editors** to
+considered. This made it especially difficult both for those proposing as well as those reviewing changes to know when a suggested change was adopted since PR comments
+could re-open a discussion. It also left a significant burden on maintainers to
 draw a line where further discussion was not possible. Such a situation leads
 to slower specification evolution and potential deadlocks.
 
+Additionally, extended GitHub discussions had become difficult to follow. Without some curation of the content, it was unclear to the those not deeply involved from the beginning to see what decisions had been made, by whom, when and why. There were also limited options for managing the often complex conversions and differing opinions that arise in technical conversations.
```

Comment:
extra "the" in `unclear to the those..`

Reply:
Thanks. (If you use the +/- button 

![image](https://github.com/user-attachments/assets/c6defcbb-12ce-49ca-94bc-53d8041014cb)

on any minor changes, I can adopt them all in one fell swoop)